apis/Google.Cloud.DevTools.Common/Google.Cloud.DevTools.Common/Source/V1/SourceContext.File.cs

```diff
@@ -30,6 +30,16 @@ public sealed partial class SourceContext
         private static Lazy<SourceContext> s_sourceContext = new Lazy<SourceContext>(OpenParseFile);
```

Comment:
It may be better now to not initialize these here and instead call `ResetAppSourceContext()` in the constructor so the initialization logic only exists in a single place.